modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -96,6 +96,71 @@ endif::alibabacloud-default,alibabacloud-customizations[]
 This command can take a few moments to run.
 ====
 
+ifdef::aws-sts[]
+.. If your cluster uses cluster capabilities to disable one or more optional components, delete the `CredentialsRequest` custom resources for any disabled components.
++
+.Example `credrequests` directory contents for {product-title} 4.12 on AWS
++
+[source,terminal]
+----
+0000_30_machine-api-operator_00_credentials-request.yaml <1>
+0000_50_cloud-credential-operator_05-iam-ro-credentialsrequest.yaml <2>
+0000_50_cluster-image-registry-operator_01-registry-credentials-request.yaml <3>
+0000_50_cluster-ingress-operator_00-ingress-credentials-request.yaml <4>
+0000_50_cluster-network-operator_02-cncc-credentials.yaml <5>
+0000_50_cluster-storage-operator_03_credentials_request_aws.yaml <6>
+----
++
+<1> The Machine API Operator CR is required. 
+<2> The Cloud Credential Operator CR is required.
+<3> The Image Registry Operator CR is required.
+<4> The Ingress Operator CR is required.
+<5> The Network Operator CR is required.
+<6> The Storage Operator CR is an optional component and might be disabled in your cluster.
```

Comment:
might be disabled in your cluster
might or can? 
This could mean that you could disable the component or it is possible the component is currently disabled.

Reply:
Will folks need instructions on how to view their credrequests directory or how to delete the CR? I assume `oc delete CredentialsRequest 0000_50_cluster-storage-operator_03_credentials_request_aws.yaml`